src/game/server/netgui.cpp

```diff
@@ -112,8 +90,8 @@ void CNetGui::OnMessage(int MsgID, void *pRawMsg, int ClientID)
 
 void CNetGui::RemoveElement(int ClientID, int Type, int NetGuiElemID)
```

Comment:
Where is function is called ? Is it not better to check compatibility when it's called instead of in the function it self ?